src/App.css

```diff
@@ -1,38 +1,133 @@
+/* CSS */
+
 .App {
-  text-align: center;
+  height: 600px;
+  overflow: scroll;
+  padding-top: 100px;
+}
+
+h1 {
+  font-family: "Libre Barcode 128 Text", cursive;
+  font-size: 30px;
+  color: white;
+  font-weight: lighter;
+  margin-left: 20px;
+}
+
+/* each info for shoes */
+
+.main_container {
+  display: flex;
+  justify-content: center;
+  /* position: relative; */
+  /* top: 85px; */
+}
+.img_container {
+  width: 80%;
+  overflow-x: scroll;
+  /* justify-content: center; */
+  display: flex;
+  scroll-behavior: smooth;
+  box-shadow: gray;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.img {
+  width: 350px;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.img:hover {
+  cursor: pointer;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+/* shoes .list */
+.listContainer {
+  display: flex;
+  justify-content: center;
+  margin-top: 50px;
+  /* position: relative; */
+  /* top: 80px; */
+  flex-wrap: wrap;
+  width: 75%;
+}
+
+.container {
   display: flex;
-  flex-direction: column;
-  align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
 }
 
-.App-link {
-  color: #61dafb;
+.infoCard {
+  width: 200px;
+  flex-wrap: wrap;
+  padding-left: 8px;
+  padding-top: 6px;
+  margin: 4px 8px;
+  border: 1px solid white;
+  transition: all 0.7s ease 0s;
+  overflow-y: scroll;
+  scroll-behavior: smooth;
+}
+
+.infoCard:hover {
+  background: rgb(252, 212, 226);
+  opacity: 0.9;
+  border-radius: 5px;
+  transform: translateY(+0.18em);
+}
+
+.name {
+  color: palevioletred;
+  font-size: 16px;
+  font-weight: bolder;
+  font-family: "Bungee Inline", cursive;
+  height: 76px;
```

Comment:
It's usually better to not make text elements be a specific height - if the text changes then this might look wrong later (if the text is too long, then other elements will overlap with the text). But it's not a big deal for this website!